setup.py

```diff
@@ -26,7 +26,7 @@
     "Topic :: Software Development :: Libraries :: Python Modules",
 ]
 URL = "https://github.com/freelawproject/seal-rookery"
-DOWNLOAD_URL = "%s/archive/%s.tar.gz" % (URL, VERSION)
+DOWNLOAD_URL = f"{URL}/archive/{VERSION}.tar.gz"
```

Comment:
```
setup.py:29:16: UP031 Use format specifiers instead of percent format
   |
27 | ]
28 | URL = "https://github.com/freelawproject/seal-rookery"
29 | DOWNLOAD_URL = "%s/archive/%s.tar.gz" % (URL, VERSION)
   |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ UP031
30 |
31 | with open("README.md", encoding="utf-8") as f:
   |
   = help: Replace with format specifiers
```

https://docs.astral.sh/ruff/rules/printf-string-formatting/